.circleci/config.yml

```diff
@@ -2,9 +2,48 @@ version: 2.1
 
 orbs:
   orb-tools: circleci/orb-tools@9.0.0
+  # Integration tests
+  secrethub: secrethub/cli@<<pipeline.parameters.dev-orb-version>>
+  aws-cli: circleci/aws-cli@0.1.22
+
+parameters:
+  run-integration-tests:
+    type: boolean
+    default: false
+  dev-orb-version:
+    type: string
+    default: "dev:latest"
+
+jobs:
+  test-integration-exec:
+    docker:
+      - image: cimg/base:stable
+    environment:
+      DOCKER_USERNAME: secrethub://company/app/docker/username
+      DOCKER_PASSWORD: secrethub://company/app/docker/password
+    steps:
+      - checkout
+      - secrethub/exec:
+          step-name: Validate if secrets have been provisioned
+          command: |
+            set -e
+            : ${DOCKER_USERNAME:?DOCKER_USERNAME is not set}
+            : ${DOCKER_PASSWORD:?DOCKER_PASSWORD is not set}
```

Comment:
Do we also want to check whether they're set to the correct value? Similar to [how we test our GitHub Action](https://github.com/secrethub/actions/blob/v0.1.0/.github/workflows/main.yml#L14)